src/ripple/rpc/impl/RPCHelpers.cpp

```diff
@@ -171,25 +252,27 @@ getAccountObjects(
             found = true;
         }
 
+        // it's possible that the returned NFTPages exactly filled the
+        // response.  Check for that condition.
+        if (i == mlimit && mlimit < limit)
```

Comment:
curious why `i == mlimit` was preferred here over than writing `0 == mlimit`

Reply:
It's been a while.  As I recall this change was motivated by one of the new unit tests (that failed before this change).  FWIW, `i` is incrementing inside the outer `for` loop, so I suspect that a comparison to `0` would not give the correct results.

Reply:
ah sorry i misread the code.